solutions/src/test/java/ru/sberbank/school/task09/RouteServiceCachedTest.java

```diff
@@ -0,0 +1,41 @@
+package ru.sberbank.school.task09;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RouteServiceCachedTest {
+    private static RouteServiceCached routeServiceCached;
+
+    @BeforeAll
+    public static void initialization() {
+        routeServiceCached = new RouteServiceCached();
+    }
+
+    @Test
+    @DisplayName("Test equals")
+    public void equalsRoutesTest() {
+        Route<City> route = routeServiceCached.getRoute("Chelyabinsk", "Moscow");
+        Route<City> route_2 = routeServiceCached.getRoute("Chelyabinsk", "Moscow");
+        Assertions.assertEquals(route.getRouteName(), route_2.getRouteName());
+        Assertions.assertIterableEquals(route.getCities(), route_2.getCities());
+    }
+
+    @Test
+    @DisplayName("Timecheck")
+    void TimeCheck() {
```

Comment:
Method name &#39;TimeCheck&#39; must match pattern &#39;^[a-z][a-z0-9][a-zA-Z0-9_]*$&#39;.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-2093252084>*